docs/more/dev-tools/repl.rst

```diff
@@ -186,15 +219,15 @@ plus ``oso``, and then use the ``Oso.repl()`` API method to start the REPL:
                 }
             }
 
-    .. group-tab:: Node
+    .. group-tab:: Node.js
 
         .. code-block:: javascript
             :caption: :fab:`node-js` app_repl.js
 
-            const { Expense, User } = require("./models");
-            const { Oso } = require("oso");
+            const { Expense, User } = require('./models');
+            const { Oso } = require('oso');
 
             const oso = new Oso();
             oso.registerClass(Expense);
             oso.registerClass(User);
-            await oso.repl();
+            oso.repl();
```

Comment:
Why did you remove `await` here?  This is still an async method.

Reply:
Because if someone copies this code into a script and runs `node path/to/that/script.js` it'll work either way, but if someone opens up the Node REPL and copies and pastes these lines in it'll blow up if they try to use a top-level `await` (unless they started Node with the `--experimental-repl-await` feature) but it'll work fine if they skip the `await`.